examples/tests/backend_driver.py

```diff
@@ -2,341 +2,46 @@
 This is used to drive many of the examples across the backends, for
 regression testing, and comparing backend efficiency.
 
-You can specify the backends to be tested either via the --backends
-switch, which takes a comma-separated list, or as separate arguments,
-e.g.
+You can specify the backends to be tested via the --backends
+switch, which takes a comma-separated list, e.g.
 
-    python backend_driver.py agg ps
+    python backend_driver.py --backends agg,ps
 
-would test the agg and ps backends. If no arguments are given, a
+would test the agg and ps backends. If --backends is not specified, a
 default list of backends will be tested.
-
-Interspersed with the backend arguments can be switches for the Python
-interpreter executing the tests. If entering such arguments causes an
-option parsing error with the driver script, separate them from driver
-switches with a --.
 """
 
-from __future__ import print_function, division
```

Comment:
~~I figured `argparser` was new in python 3.2, so the script can't be run with python 2 anyway.~~ It looks like `argparser` is in 2.7, will put these lines back.